content/en/docs/concepts/cluster-administration/system-metrics.md

```diff
@@ -152,6 +152,21 @@ Once a pod reaches completion (has a `restartPolicy` of `Never` or `OnFailure` a
 The metrics are exposed at the HTTP endpoint `/metrics/resources` and require the same authorization as the `/metrics`
 endpoint on the scheduler. You must use the `--show-hidden-metrics-for-version=1.20` flag to expose these alpha stability metrics.
 
+## Disabling metrics
+
+You can explicitly turn off metrics via CLI flag `--disabled-metrics`. This may be desired if, for example, a metric is causing a performance problem. The input is a list of disabled metrics (i.e. `--disabled-metrics=metric1,metric2`).
+
+## Metric Cardinality Enforcement
+
+Metrics with unbounded dimensions could cause memory issues in the components they instrument. To solve this problem, we introduce a flag `--allow-label-value` to  dynamically configure an allow-list of label values for a metric.
```

Comment:
Terms like “introduce” are good for blogs and release notes; in concept docs we assume the reader is thinking about the feature in-use, so we use different wording such as
```suggestion
Metrics with unbounded dimensions could cause memory issues in the components they
instrument. To limit resource use, you can use the `--allow-label-value` command line
option to dynamically configure an allow-list of label values for a metric.
```

Reply:
Got you. changed.